language/Spanish/strings.po

```diff
@@ -114,19 +114,19 @@ msgstr "Fabricante\t%1$-.100s\n"
 #: src/frontend/mame/ui/selgame.cpp:1028
 #, c-format
 msgid "Driver is Clone of\t%1$-.100s\n"
-msgstr "Controlador es un clon de\t%1$-.100s\n"
+msgstr "El controlador es un clon de\t%1$-.100s\n"
 
 #: src/frontend/mame/ui/selgame.cpp:1030
 msgid "Driver is Parent\t\n"
 msgstr "Es un controlador padre\t\n"
 
 #: src/frontend/mame/ui/selgame.cpp:1033
 msgid "Analog Controls\tYes\n"
-msgstr ""
+msgstr "Controles analógicos"
 
 #: src/frontend/mame/ui/selgame.cpp:1035
 msgid "Keyboard Inputs\tYes\n"
-msgstr ""
+msgstr "Entradas de teclado"
```

Comment:
As above, this isn't going to work right without something for the right-hand column and the final newline.